src/v2/guide/components-slots.md

```diff
@@ -51,25 +53,101 @@ O incluso otros componentes:
 
 Si `<navigation-link>` **no** contiene un elemento `<slot>`, cualquier contenido que se le pase será simplemente descartado.
 
-## Slots Nombrados
+## Scope de compilación
+
+Cuando desee utilizar datos dentro de un slot, como en:
+
+```html
+<navigation-link url="/profile">
+  Inició sesión como {{ nombre.de.usuario }}
+</navigation-link>
+```
+
+Ese slot tiene acceso a las mismas propiedades de instancia (es decir, el mismo `scope`) que el resto de la plantilla. El slot _no_ tiene acceso al "scope" de `<navigation-link>`. Por ejemplo, intentar acceder a `url` no funcionaría:
+
+```html
+<navigation-link url="/profile">
+  Haciendo clic aquí te enviará a: {{ url }}
+  <!--
+  La `url` será indefinida, porque este contenido se pasa _a_ <navigation-link>, 
```

Comment:
Is `indefinida` a translation for `undefined`? If so, we should keep `undefined` as it most likely refer to the type, not the word.

Reply:
Sure!